### ISSUE TITLE
Udpate contributing.rst

### ISSUE THREAD

**Issue by geigerj** [GoogleCloudPlatform/google-cloud-python#3300]
Information in [contributing.rst](https://github.com/GoogleCloudPlatform/google-cloud-python/blob/master/CONTRIBUTING.rst) referring to `tox` or to the `system_tests` directory no longer works after #3146.

**Comment by dhermes** [comment]
Closing as a duplicate of #3205, thanks for taking the time to file @geigerj!

Last I recall, @lukesneeringer was on the hook for this, though maybe that's no longer true.

**Comment by lukesneeringer** [comment]
I am still on the hook for this. :-)
